apps/web/features/customization/components/widget-preview.tsx

```
@@ -28,96 +27,82 @@ export const WidgetPreview = ({ data }: WidgetPreviewProps) => {
 								</div>
-								<button
-									onClick={() => setIsOpen(false)}
-									className="text-white/80 hover:text-white"
-								>
-									<X className="h-5 w-5" />
-								</button>
 							</div>
+							<button
+								onClick={() => setIsOpen(false)}
+								className="text-white/80 hover:text-white"
+							>
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)